person.rb

```diff
@@ -1,14 +1,21 @@
-class Person
+require './nameable'
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(age, name: 'Unknown', parent_permission: true)
+    super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
   end
```

Comment:
- Good job adding decorator classes to your project so far, but I have this error when running your project, a likely error could be that you are assigning the default value for the name wrongly. I suggest using `name='Unknown'`. Please fix this 🙏 

Your project:
![2f23rf](https://user-images.githubusercontent.com/73607512/230991202-d82cce76-4e72-4e6b-9537-a2e08f6b6603.PNG)